specification/storage/resource-manager/Microsoft.Storage/stable/2021-06-01/storage.json

```diff
@@ -2685,6 +2685,10 @@
           "type": "boolean",
           "description": "Allow or disallow cross AAD tenant object replication. The default interpretation is true for this property."
         },
+        "defaultToOAuthAuthentication": {
+          "type": "boolean",
+          "description": "A boolean flag which indicates whether the default authentication is OAuth or not. The default interpertation is false for this property."
```

Comment:
what's the difference of set this property to true and false?
IS there any feature spec?